project/api/models.py

```diff
@@ -128,10 +129,6 @@ def clean(self, *args, **kwargs):
 
         super(Territory, self).clean(*args, **kwargs)
 
-    def save(self, *args, **kwargs):
-        self.full_clean()
```

Comment:
The `test_model_can_not_create_territory` failure is happening because of the deletion of this line.